### PR TITLE
Propagate disabled state from items to buttons

### LINKED ISSUE #13: Disabling a root-level MenuItem should also disable the button
Currently, only the `vaadin-context-menu-item` inside the button is disabled. The button still looks and feels clickable, although the actions are not executed.

### PR DESCRIPTION
**Pull request by pekam**
Fix #13

### CODE REVIEW

File: vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java [hunk 1]
```diff
@@ -66,11 +66,25 @@ public MenuBarTestPage() {
                 e -> menuBar.setOpenOnHover(!menuBar.isOpenOnHover()));
         openOnHoverButton.setId("toggle-open-on-hover");
 
-        NativeButton setWidthButton = new NativeButton("set width 150px",
-                e -> setWidth("150px"));
+        NativeButton setWidthButton = new NativeButton("set width 150px", e -> {
```
Comment (yuriy-fix) [inline vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java]: `set width 150px` != `setWidth("140px")` 🤔

File: vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java [hunk 2]
```diff
@@ -66,11 +66,25 @@ public MenuBarTestPage() {
...
+            setWidth("140px");
+            menuBar.getElement().callFunction("notifyResize");
+        });
         setWidthButton.setId("set-width");
 
+        NativeButton resetWidthButton = new NativeButton("reset width", e -> {
+            setWidth("initial");
```
Comment (yuriy-fix) [inline vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java]: Default value for the `width` property is `auto`.

File: vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java (hunk 1 above)
Comment (pekam) [reply vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java]: done

File: vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java (hunk 2 above)
Comment (pekam) [reply vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java]: done

**Pull request merged**